core/src/main/java/org/eclipse/hono/util/Constants.java

```diff
@@ -88,6 +88,12 @@
      * attachments.
      */
     public static final String KEY_CLIENT_PRINCIPAL = "CLIENT_PRINCIPAL";
+    /**
+     * They key that an authenticated client of a protocol adapter (representing a device)
+     * is stored under in a {@code ProtonConnection}'s attachments.
+     */
+    public static final String KEY_CLIENT_DEVICE = "CLIENT_DEVICE";
```

Comment:
this seems to be very specific to the AMQP adapter, i.e. it is of no relevance to any other component, is it? So, maybe you can find a more appropriate place for this constant?

Reply:
yes sure..